File: scratch_fix_links.py
import re

with open('index.html', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace trendyol links in product cards with product.html
content = re.sub(r'href="https://www\.trendyol\.com/[^"]+"', 'href="product.html"', content)

with open('index.html', 'w', encoding='utf-8') as f:
    f.write(content)
print("Updated links in index.html")

File: scratch_fix_links.js
const fs = require('fs');

let content = fs.readFileSync('index.html', 'utf8');

// Replace trendyol links in product cards with product.html
content = content.replace(/href="https:\/\/www\.trendyol\.com\/[^"]+"/g, 'href="product.html"');

fs.writeFileSync('index.html', content, 'utf8');
console.log("Updated links in index.html");
